src/service/extension-location.js

```diff
@@ -49,13 +49,19 @@ export function calculateExtensionScriptUrl(location, extensionId, isLocalDev) {
  * @param {string} entryPoint
  * @param {boolean=} isLocalDev
  * @param {boolean=} opt_rtv
+ * @param {boolean=} opt_minified
  * @return {string}
  */
 export function calculateEntryPointScriptUrl(
-    location, entryPoint, isLocalDev, opt_rtv) {
+    location, entryPoint, isLocalDev, opt_rtv, opt_minified) {
```

Comment:
Looks like [that var refers to](https://github.com/ampproject/amphtml/blob/401aaafe7c7fde7c994a32e67a23bbd6e4a7dfeb/build-system/tasks/runtime-test.js#L27) `test-server.js`. Perhaps we should consolidate though.